lib/kernel.coffee

```diff
@@ -15,7 +15,7 @@ class Kernel
         console.log 'Kernel spec:', @kernelSpec
         console.log 'Kernel configuration:', @config
         console.log 'Kernel configuration file path:', @configPath
-        @language = @kernelSpec.grammarLanguage
+        @language = @kernelSpec.language
```

Comment:
`@language` was always `undefined`. Hence the hardcoded ipython kernel couldn't ever work.

I think we can remove it because nobody reported an issue 😉 .